src/actions/index.js

```diff
@@ -9,10 +9,18 @@ export const DECREMENT = 'DECREMENT';
 // logic of updating the central Redux store. That
 // is left to the reducer(s).
 
-export const increment = () => {
+export const increment = (num) => {
   // Fill in this function
+  return {
+    type: INCREMENT,
+    payload: num
+  }
 };
```

Comment:
```
export const increment = (num) => {
  // Fill in this function
  return {
    type: INCREMENT,
    payload: num
  }
};
```

Because you only need your action's type in your reducer, you don't need any other properties on it.

ie. You could have written it:

```
export const increment = () => {
  // Fill in this function
  return { type: INCREMENT }
};
```